composer.json

```diff
@@ -22,19 +22,14 @@
         "news": "nntp://lists.typo3.org"
     },
     "require": {
-        "typo3/cms-backend": "9.5.*@dev",
-        "typo3/cms-core": "9.5.*@dev",
-        "typo3/cms-extbase": "9.5.*@dev",
-        "typo3/cms-extensionmanager": "9.5.*@dev",
-        "typo3/cms-filelist": "9.5.*@dev",
-        "typo3/cms-fluid": "9.5.*@dev",
-        "typo3/cms-frontend": "9.5.*@dev",
-        "typo3/cms-install": "9.5.*@dev",
-        "typo3/cms-recordlist": "9.5.*@dev"
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "9.5.x-dev"
-        }
+        "typo3/cms-backend": "9.5.*",
```

Comment:
Thanks a lot for taking the time to create this pull request.

However this change isn't required, as Composer will happily install a stable TYPO3 version 9.5.0 with the existing version constraint.